### PR TITLE
bottles: cleanup

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/bottles/default.nix
Comment (psydvl) [inline pkgs/applications/misc/bottles/default.nix:104]: There is no updater in fetchFromGitHub https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchgithub/default.nix, only downloader

**Review by psydvl — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/bottles/default.nix
Comment (SuperSandro2000) [reply pkgs/applications/misc/bottles/default.nix:107]: There is. How do you think all the other packages get updated? You can use the script in nixpkgs or nix-update. No need to add a custom vendored script.

Comment (psydvl) [reply pkgs/applications/misc/bottles/default.nix:107]: I see only automatic updates for packages with `passthru.UpdateScript`
You can look into last update for this package: https://github.com/NixOS/nixpkgs/pull/169250; it runs using https://github.com/ryantm/nixpkgs-update which follows `passthru.UpdateScript`

Comment (psydvl) [reply pkgs/applications/misc/bottles/default.nix:107]: Look into 6.4.3.2. passthru.updateScript https://nixos.org/manual/nixpkgs/stable/#special-variables

Comment (OPNA2608) [reply pkgs/applications/misc/bottles/default.nix:107]: > You can use the script in nixpkgs

Which still needs a `passthru.updateScript`… There was a new release 4 days ago. I ran `nix-shell maintainers/scripts/update.nix --argstr package bottles` after you merged this PR and no update is performed.

Comment (SuperSandro2000) [reply pkgs/applications/misc/bottles/default.nix:107]: > I see only automatic updates for packages with passthru.UpdateScript

Thats wrong. None of my packages have that and still get auto updates from r-rynatm.

**Pull request merged**
